### ISSUE TITLE
Bug: Litestar changing root logger by default

### ISSUE THREAD

**Issue by arthurazs** [litestar-org/litestar#2862]
### Description

Litestar changes the root logger configuration by default.

My current workaround is to set Litestar root logger to an empty dict:

`app = Litestar(logging_config=LoggingConfig(root={}))`

### MCVE

```python
import logging

from litestar import Litestar


logging.basicConfig(level=logging.INFO, format="CRAZY LOG %(message)s")
logger = logging.getLogger(__name__)

logger.info("Logging from lite_app before creating the litestar app")
app = Litestar()
logger.info("Logging from lite_app after creating the litestar app")
```

### Steps to reproduce

Execute above code, expected output:

```bash
CRAZY LOG Logging from lite_app before creating the litestar app
CRAZY LOG Logging from lite_app after creating the litestar app
```

Current output:

```
CRAZY LOG Logging from lite_app before creating the litestar app
INFO - 2023-12-08 20:40:08,708 - __main__ - example - Logging from lite_app after creating the litestar app
```

### Litestar Version

2.3.2

### Platform

- [X] Linux
- [ ] Mac
- [X] Windows
- [ ] Other (Please specify in the description above)

---
> [!NOTE]  
> While we are open for sponsoring on [GitHub Sponsors](https://github.com/sponsors/litestar-org/) and 
> [OpenCollective](https://opencollective.com/litestar), we also utilize [Polar.sh](https://polar.sh/) to engage in pledge-based sponsorship.
>
> Check out all issues funded or available for funding [on our Polar.sh dashboard](https://polar.sh/litestar-org)
> * If you would like to see an issue prioritized, make a pledge towards it!
> * We receive the pledge once the issue is completed & verified
> * This, along with engagement in the community, helps us know which features are a priority to our users.

<a href="https://polar.sh/litestar-org/litestar/issues/2862">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/litestar-org/litestar/issues/2862/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/litestar-org/litestar/issues/2862/pledge.svg">
</picture>
</a>

**Comment by EM51641** [comment]
I have the exact same problem.

**Comment by arthurazs** [comment]
> I have the exact same problem.

While we wait for the fix, my workaround is `app = Litestar(logging_config=LoggingConfig(root={}))`

**Comment by provinzkraut** [comment]
Closed by #2969.
